toolbox-index.html

```diff
@@ -8,7 +8,7 @@
   <script src="https://cdn.polyfill.io/v2/polyfill.js?features=String.prototype.includes,Array.prototype.includes,Array.prototype.find,CustomEvent"></script>
 </head>
 
-<body>
+<body data-spy="scroll" data-target="#intermediate-nav" data-offset="0">
```

Comment:
Is it not a bit excessive ? 😅

Reply:
Not necessarily, not sure if it's working by injecting these attributes with JS.